### PR TITLE
Update projects.yml

### PR DESCRIPTION
**Pull request by ikitommi**
* reitit does async (`reitit-http`, `reitit-pedestal`)
* reitit does rest (`reitit-ring`, `reitit-http`, `reitit-pedestal`)
* reitit does web (`reitit-ring`, `reitit-http`, `reitit-pedestal`)
* compojure-api is Web Frameworks
* jsonista
* spec-tools
* schema-tools
* schema-viz

### CODE REVIEW

File: projects.yml [hunk 1]
```diff
@@ -4350,3 +4350,27 @@ coast:
...
   categories: [Web Frameworks]
   platforms: [clj]
+
+jsonista:
+  name: Jsonista
+  url: https://github.com/metosin/jsonista
+  categories: [JSON Parsers]
+  platforms: [clj]
+
+spec-tools:
```
Comment (weavejester) [inline projects.yml]: Can you change this to `spec_tools`

File: projects.yml [hunk 2]
```diff
@@ -4350,3 +4350,27 @@ coast:
...
+  name: Spec-Tools
+  url: https://github.com/metosin/spec-tools
+  categories: [Spec]
+  platforms: [clj, cljs]
+
+schema-tools:
```
Comment (weavejester) [inline projects.yml]: Can you change this to `schema_tools`

File: projects.yml [hunk 3]
```diff
@@ -4350,3 +4350,27 @@ coast:
...
+  name: Schema-Tools
+  url: https://github.com/metosin/schema-tools
+  categories: [Schema]
+  platforms: [clj, cljs]
+
+schema-viz:
```
Comment (weavejester) [inline projects.yml]: Can you change this to `schema_viz`

**Comment by ikitommi** [comment]
Fixed those.

**Pull request merged**